Develop an SQL schema for SOPs

```diff
@@ -0,0 +1,4351 @@
+
+--
+-- TOC entry 2828 (class 2606 OID 18261)
+-- Name: Small Molecule Section Small Molecule Section_SMF_ID_REFS_fkey; Type: FK CONSTRAINT; Schema: MzTab-M; Owner: postgres
+--
+
+ALTER TABLE ONLY "MzTab-M"."Small Molecule Section"
+    ADD CONSTRAINT "Small Molecule Section_SMF_ID_REFS_fkey" FOREIGN KEY ("SMF_ID_REFS") REFERENCES "MzTab-M"."Small Molecule Feature (SMF) Section"("SMF_ID") NOT VALID;
+
+
+-- Completed on 2020-10-30 17:54:26
+
+--
+-- PostgreSQL database dump complete
+--
+
+--
+-- PostgreSQL database dump
+--
+
+-- Dumped from database version 12.3
+-- Dumped by pg_dump version 12.3
+
+-- Started on 2020-10-30 17:12:34
+
+SET statement_timeout = 0;
+SET lock_timeout = 0;
+SET idle_in_transaction_session_timeout = 0;
+SET client_encoding = 'UTF8';
+SET standard_conforming_strings = on;
+SELECT pg_catalog.set_config('search_path', '', false);
+SET check_function_bodies = false;
+SET xmloption = content;
+SET client_min_messages = warning;
+SET row_security = off;
+
+DROP DATABASE "MFA analysis";
+--
+-- TOC entry 3134 (class 1262 OID 17718)
+-- Name: MFA analysis; Type: DATABASE; Schema: -; Owner: postgres
+--
+
+CREATE DATABASE "MFA analysis" WITH TEMPLATE = template0 ENCODING = 'UTF8' LC_COLLATE = 'German_Germany.1252' LC_CTYPE = 'German_Germany.1252';
+
+
+ALTER DATABASE "MFA analysis" OWNER TO postgres;
+
+\connect -reuse-previous=on "dbname='MFA analysis'"
+
+SET statement_timeout = 0;
+SET lock_timeout = 0;
+SET idle_in_transaction_session_timeout = 0;
+SET client_encoding = 'UTF8';
+SET standard_conforming_strings = on;
+SELECT pg_catalog.set_config('search_path', '', false);
+SET check_function_bodies = false;
+SET xmloption = content;
+SET client_min_messages = warning;
+SET row_security = off;
+
+SET default_tablespace = '';
+
+SET default_table_access_method = heap;
+
+--
+-- TOC entry 209 (class 1259 OID 17752)
+-- Name: MDVFragments_from_excel; Type: TABLE; Schema: public; Owner: postgres
+--
+
+CREATE TABLE public."MDVFragments_from_excel" (
+    met_id text NOT NULL,
+    formula text NOT NULL,
+    elements text[] NOT NULL,
+    positions integer[] NOT NULL,
+    id integer NOT NULL
+);
+
+
+ALTER TABLE public."MDVFragments_from_excel" OWNER TO postgres;
+
+--
+-- TOC entry 208 (class 1259 OID 17750)
+-- Name: MDVFragments_id_seq; Type: SEQUENCE; Schema: public; Owner: postgres
+--
+
+CREATE SEQUENCE public."MDVFragments_id_seq"
+    AS integer
+    START WITH 1
+    INCREMENT BY 1
+    NO MINVALUE
+    NO MAXVALUE
+    CACHE 1;
+
+
+ALTER TABLE public."MDVFragments_id_seq" OWNER TO postgres;
+
+--
+-- TOC entry 3135 (class 0 OID 0)
+-- Dependencies: 208
+-- Name: MDVFragments_id_seq; Type: SEQUENCE OWNED BY; Schema: public; Owner: postgres
+--
+
+ALTER SEQUENCE public."MDVFragments_id_seq" OWNED BY public."MDVFragments_from_excel".id;
+
+
+--
+-- TOC entry 207 (class 1259 OID 17743)
+-- Name: MFA model metabolites ; Type: TABLE; Schema: public; Owner: postgres
+--
+
+CREATE TABLE public."MFA model metabolites " (
```

Comment:
Please remove all white spaces or replace white spaces with and underscore `_`

Reply:
done